src/Microsoft.Azure.SignalR.Common/ServiceConnections/StrongServiceConnectionContainer.cs

```diff
@@ -35,10 +35,17 @@ public override Task HandlePingAsync(PingMessage pingMessage)
 
         public override Task StopAsync()
         {
-            var stopOndemandConnectionsTask =  Task.WhenAll(_onDemandServiceConnections.Select(c => c.StopAsync()));
+            var stopOndemandConnectionsTask = Task.WhenAll(_onDemandServiceConnections.Select(c => c.StopAsync()));
             return Task.WhenAll(stopOndemandConnectionsTask, base.StopAsync());
         }
 
+        public override Task ShutdownAsync(TimeSpan timeout)
+        {
+            StartShutdown();
```

Comment:
no need to expose `StartShutdown`? call `base.ShutdownAsync` is enough?

Reply:
`StartShutdown` is used by both `Shutdown` and `StopAsync`.